src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (val: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pages = getNumbers(1, Math.ceil(total / perPage));
+  const handlePrev = () => {
+    if (currentPage === 1) {
+      return;
+    }
+
+    onPageChange(currentPage - 1);
+  };
+
+  const handleNext = () => {
+    if (currentPage === pages.length) {
+      return;
+    }
+
+    onPageChange(currentPage + 1);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={handlePrev}
```

Comment:
It is a good practice to prevent the default action of anchor tags when they are used as buttons. You should call `event.preventDefault()` inside the `handlePrev` function to prevent the page from jumping to the top when the link is clicked.